tests/locator_test.go

```diff
@@ -91,7 +90,11 @@ func TestLocator(t *testing.T) {
 					return asBool(t, v)
 				}
 				require.False(t, result(), "should not be clicked first")
-				p.Locator("#link", nil).DispatchEvent("click", tb.toGojaValue("mouseevent"), nil)
+				opts := &common.FrameDispatchEventOptions{
```

Comment:
Nit: Could you have used `common.NewFrameDispatchEventOptions()` instead of declaring the options like this?

Reply:
Sure, I thought about that. But it requires setting a timeout by the constructor. Since other tests don't use a timeout (and do it in the timeout tests below), I wanted to keep empty. WDYT?

Reply:
No strong opinion. My concern is whether it will work for everyone if the timeout is `0`.

Reply:
You mean setting a `0` timeout using the constructor? It's been working so far that way. And the current code also does that.

Reply:
Yeah, it'll be defaulting to `0`. I'm happy with how it is now, it's not crucial to use the New method.

Reply:
Done: [eca3338](https://github.com/grafana/xk6-browser/pull/1193/commits/eca33389a91ddaac3407df43745f5587eb3e3fc9).